test/check-machines-snapshots

```diff
@@ -469,6 +482,101 @@ class TestMachinesSnapshots(machineslib.VirtualMachinesCase):
         self.assertEqual("no", m.execute("virsh snapshot-info --domain subVmTest1 --snapshotname snapshotB | grep 'Current:' | awk '{print $2}'").strip())
         self.assertEqual("yes", m.execute("virsh snapshot-info --domain subVmTest1 --snapshotname snapshotC | grep 'Current:' | awk '{print $2}'").strip())
 
+    def testMemoryLocation(self):
+        b = self.browser
+        m = self.machine
+
+        if not supportsExternalSnapshots(m.image):
+            raise unittest.SkipTest("external snapshots not supported")
+
+        self.createVm("subVmTest1")
+
+        self.login_and_go("/machines")
+        self.waitPageInit()
+        self.waitVmRow("subVmTest1")
+        b.wait_in_text("#vm-subVmTest1-system-state", "Running")
+        self.goToVmPage("subVmTest1")
+
+        # Open dialog
+        b.click("#vm-subVmTest1-add-snapshot-button")
+        b.set_input_text("#snapshot-create-dialog-name", "snap-1")
+
+        # Check default
+        b.wait_val("#snapshot-create-dialog-memory-location input", "/var/lib/libvirt/images")
+
+        # Check validation
+        b.set_input_text("#snapshot-create-dialog-memory-location input", "")
+        b.wait_text("#snapshot-create-dialog-memory-location .pf-v5-c-helper-text__item-text",
+                    "Memory save location can not be empty")
+
+        # Check creation of new directories
+        b.set_input_text("#snapshot-create-dialog-memory-location input", "/var/lib/libvirt/new")
+        b.wait_text("#snapshot-create-dialog-memory-location li button", 'Create "/var/lib/libvirt/new"')
+        b.click("#snapshot-create-dialog-memory-location li button")
+
+        # Check available space detection
+        b.wait_in_text("#snapshot-create-dialog-memory-location .pf-v5-c-helper-text__item-text",
+                       "available at this location")
+        b.wait_in_text("#snapshot-create-dialog-memory-location .pf-v5-c-helper-text__item-text",
+                       "are needed to save the memory state of the machine.")
```

Comment:
Please check that this has variant "default"

Reply:
Done. We have to check that it is not a warning.